### DISCUSSION TITLE
Allow overwrite cassete values

### DISCUSSION THREAD

**Discussion by LuanGB** [vcr/vcr#862] [Ideas]
It would be very nice to have the option of replacing values of a cassete for some cases.
When setting up a test, some requests might include nested resources with sequential data (like an id). So, when choosing witch cassete to use for a given request, having a way of replacing the values with the previous created record's data would be very useful.

I see myself using this kind of like this:
```ruby
    let(:item1) { create :item }
    let(:item2) { create :item } # these must have different ids
    let(:order) { create :order, items: [ item1, item2 ] }
...
    VCR.use_cassette("synopsis") do |cassete|
      cassete.attributes[:body][:order] = order.id
      # or
      cassete.dig(:body, :order, :items, 0, :id) = item1.id
      cassete.dig(:body, :order, :items, 0, :id) = item2.id

      response = Net::HTTP.get_response(URI("mydomain/orders/#{order.id}"))
      assert_match /Example domains/, response.body
    end
...
```

**Comment by dmitry** [comment]
@LuanGB even now you can use the following syntax:

```ruby
VCR.use_cassette("synopsis", erb: {order_id: order.id}) do |cassete|
``` 

But for sure it might be nice idea to add possibility to change cassette body and some other request/response fields (headers, queries, etc).
